tests/tt_metal/tt_metal/unit_tests_common/watcher/test_noc_sanitize.cpp

```diff
@@ -122,7 +164,16 @@ TEST_F(WatcherFixture, TestWatcherSanitize) {
         GTEST_SKIP();
 
     CheckHostSanitization(this->devices_[0]);
-    for (Device* device : this->devices_) {
-        this->RunTestOnDevice(RunTest, device);
-    }
+
+    // Only run on device 0 because this test takes down the watcher server.
+    this->RunTestOnDevice(RunTest, this->devices_[0]);
+}
+
+TEST_F(WatcherFixture, TestWatcherSanitizeEth) {
+    // Skip this test for now until we add a way to recover from the hang.
+    log_info(LogTest, "Skip this test until we can recover from the hang on an eth core.");
```

Comment:
This is issue will be a bit more involved to fix, filed a separate issue here: #5397 

We'll put that fix in as a separate PR, so just disable for now.